### DISCUSSION TITLE
Vorschläge für Tags

### DISCUSSION THREAD

**Discussion by Schnellzug414** [Traewelling/traewelling#2900] [Ideas / Feature Requests]
Hi, es wäre echt praktisch wenn ihr bei den Tags auch eine Spalte für den Taufnamen des Triebzuges oder den Zugnamen (z.B. Wattenmeer, Blauer Enzian...) hinzufügen würdet

**Comment by kpleiss** [comment]
Gute Idee. Das taugt auch für die Tz Nummer wie z.b. bei ICE oder IC2.

**Reply by marhei** [reply]
Für Fahrzeugnummern gibt es doch mit `trwl:vehicle_number` bereits ein vordefinierten Tag.

**Comment by MrKrisKrisu** [comment]
> Taufnamen des Triebzuges oder den Zugnamen

Was spricht dagegen, es in `trwl:vehicle_number` (Fahrzeugnummer) zu speichern?

**Comment by kpleiss** [comment]
Schreibt man da nicht lieber die 93 80 4812 xxx-x D-DB rein und dann in ein anderes Tag Triebzug 9023 oder Taufname Beispielstadt..?

**Reply by MrKrisKrisu** [reply]
Es ist nicht vorgegeben, was man wo reinschreiben kann. Im Prinzip sind das nur Vorschläge.

Technisch gesehen (über die API) kann jeder User auch beliebige Felder nutzen, das ist nur leider aktuell nicht im Frontend umgesetzt, dass man hier noch eingeschränkt ist.

**Reply by kpleiss** [reply]
Alles klar, danke für die Klarstellung! :-)
